goaviatrix/tags.go

```diff
@@ -82,7 +82,7 @@ func (c *Client) GetTagsMap(tags *Tags) (map[string]string, error) {
 	}
 
 	var resp TagAPIResp
-	err := c.GetAPI(&resp, data["Action"], data, BasicCheck)
+	err := c.GetAPI(&resp, data["action"], data, BasicCheck)
```

Comment:
GetTagsMap is not being used anymore because tags are gotten from list_vpcs_summary